nerdlandbot/__main__.py

```diff
@@ -46,9 +46,13 @@
 bot.load_extension("nerdlandbot.commands.poll")
 bot.load_extension("nerdlandbot.commands.purger")
 bot.load_extension("nerdlandbot.commands.kerk")
+
+bot.load_extension("nerdlandbot.commands.dad_joke")
```

Comment:
probably a funky effect from merging develop back into branch but this shouldn't be here I think.

Reply:
I'm sorry, this is the first time I'm developing with other people :-).
Nut sure what I'm supposed to do then when I get "Conflicts". 
I have merged both versions but that seems not to be what is expected ?

Reply:
Ok, I got help in the discord. I've removed line 50 because dad_joke is now part of another cog.